src/cashlib/cashlib.cpp

```diff
@@ -58,6 +58,28 @@ struct ForkDeploymentInfo
 };
 struct ForkDeploymentInfo VersionBitsDeploymentInfo[Consensus::MAX_VERSION_BITS_DEPLOYMENTS];
 
+// Must match the equivalent object in calling language code
+typedef enum {
+    AddrBlockchainBCH = 1,
+    AddrBlockchainBCHtestnet = 2,
+    AddrBlockchainBCHregtest = 3,
+    AddrBlockchainUnlimited = 4,
```

Comment:
why not calling it `AddrBlockchainNol` ?

Reply:
done